src/comm/MockLink.cc

```diff
@@ -1576,14 +1576,8 @@ bool MockLink::_handleRequestMessage(const mavlink_command_long_t& request, bool
     switch ((int)request.param1) {
     case MAVLINK_MSG_ID_COMPONENT_INFORMATION:
         if (_firmwareType == MAV_AUTOPILOT_PX4) {
-            switch (static_cast<int>(request.param2)) {
-            case COMP_METADATA_TYPE_VERSION:
-                _sendVersionMetaData();
-                return true;
-            case COMP_METADATA_TYPE_PARAMETER:
-                _sendParameterMetaData();
-                return true;
-            }
+            _sendVersionMetaData();
```

Comment:
Shouldn't this be changed to `_sendGeneralMetaData`?

Reply:
Indeed, updated